inst/doc/custom_data_sources.Rmd

```diff
@@ -0,0 +1,80 @@
+---
```

Comment:
I still don't get why this file is in two places, seems fishy to me.  Someone with more vignette experience can explain it to me.